templates/validator/charts.html

```diff
@@ -169,7 +169,60 @@
                         chart: {
                             type: 'column',
                             height: '500px',
-                            pointInterval: 24 * 3600 * 1000
+                            pointInterval: 24 * 3600 * 1000,
+                            events: {
+                                load: function () {
+                                    var chart = this,
+                                        btnWidth = 144,
+                                        btnHeight = 30,
+                                        btnX = chart.plotLeft,
+                                        btnY = chart.chartHeight - btnHeight - 10; // 10px margin from the bottom
+                                        normalState = {
+                                            fill: '#FFAA31'
+                                        },
+                                        hoverState = {
+                                            fill: '#F29A00',
+                                        }
+                                    callback = function () {
+                                        var baseURL = window.location.protocol + "//" + window.location.host;
+                                        var urlParts = window.location.href.split("/");
+                                        var lastPart = urlParts[urlParts.length - 1];
+                                        var validatorId = lastPart.split("#")[0];
+
+                                        // Build the API URL
+                                        var apiURL = baseURL + "/dashboard/data/allbalances?validators=" + validatorId;
+
+                                        fetch(apiURL)
```

Comment:
This does not include proper error handling. Forcing `apiURL` to always return an error will show an empty graph without indication to the user that something is broken (once the user presses the button):
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/1b0cd7d6-da48-472a-8355-25ccfdb745e9)